tests/README.txt

```diff
@@ -1,9 +1,22 @@
-rename tests/config.ini.changeme to tests/config.ini and
-fill the variable (dbname should different from the one used
-in production.
+Create a file named .env.test at the root of the project containing:
+
+```yaml
+APP_ENV=dev
+
+APP_DEBUG=0
+APP_SECRET=1a2e701e2de1f2b4eb9c320e23ed5cf2
+
+DATABASE_URL=mysql://<dbuser>:<dbpass>@<dbhost>:<dbport>/<dbname>
+DATABASE_PREFIX=
+
+MAILER_URL=null://localhost
+```
+
+!IMPORTANT <dbname> must be different from the production DB name.
+<dbuser> should be able to create a database.
 
 run tests like the following:
 ./vendor/bin/simple-phpunit --bootstrap tests/bootstrap.php tests/
 
```

Comment:
**Si je n'installe pas composer avec apt, j'ai l'erreur :** 
<pre>sh: 1: composer: not found
sh: 1: composer: not found</pre>

**NB1 :** j'utilise habituellement composer de cette façon : 
<pre>php /chemin/vers/composer.phar</pre>
J'ai créé un alias <pre> alias composer="php /chemin/vers/composer.phar</pre>

La commande composer -V donne : 
<pre>Composer version 2.0.11 2021-02-24 14:57:23</pre>
Donc OK.
Mais pour les tests unitaires, ça ne suffit pas.

**NB2 :** j'ai bien lancé composer install, est composer a bien été ajouté dans mon dossier vendor en suivant les instructions du composer.json, mais ça ne suffit pas.
J'ai même lancé un composer update : pareil

**S'il faut que composer soit installé au niveau du système, il faut le noter.**

Une fois composer installé via apt : les tests unitaires fonctionnent parfaitement